velox/exec/Spiller.h

```diff
@@ -70,27 +110,32 @@ class Spiller {
             pool,
             spillMappedMemory()),
         pool_(pool),
-        executor_(executor) {}
+        executor_(executor) {
+    // TODO: add to support kHashJoin type later.
+    VELOX_CHECK_NE(
```

Comment:
It would be nice to move the implementation to .cpp file. It helps with incremental build times.

Reply:
Good point. Thanks!